packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.tsx

```diff
@@ -84,23 +110,34 @@ const ChangesTab = (props: ChangesTabProps) => {
           },
           {
             id: "changes-table-actions",
-            width: 62,
+            width: 100,
             Cell: (props: CellProps<Changeset>) => {
               const changeset = props.data[props.row.index];
+              const className = canCreateVersion(changeset)
+                ? ""
+                : "iac-create-version-icon-hidden";
               return (
                 <>
-                  {canCreateVersion(changeset) && (
+                  <ButtonGroup className="iac-changes-tab-actions-btn-group">
                     <IconButton
                       onClick={() => {
                         setCurrentChangeset(changeset);
                         setIsCreateVersionModalOpen(true);
                       }}
                       title={stringsOverrides.createNamedVersion}
                       styleType="borderless"
+                      className={className}
```

Comment:
you want to hide it or dont render it instead?
oh.. probably if you dont render it, it will get content movement.. hmm.. So maybe its ok..

Reply:
yes, if I don't render then getting distorted view for action buttons- 


![image](https://github.com/iTwin/admin-components-react/assets/148448690/2939f3ad-8b90-4a5d-b06b-b1fdeb093e66)